lms/envs/common.py

```diff
@@ -2050,6 +2055,9 @@
 
     # Needed whether or not enabled, due to migrations
     'badges',
+
+    # Enables default site and redirects
+    'django_sites_extensions',
```

Comment:
If this package is dependent on django.contrib.redirects we should note it in the comments